Update Organization Permissions Documentation (#297)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/settings/access.mdx

- [Workspace-level permissions](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-permissions) can be granted to an individual team on a particular workspace. These permissions can be managed on the workspace by anyone with admin access to the workspace.
- In addition, some [organization-level permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-permissions) can be granted to a team which apply to every workspace in the organization. For example, the
  [manage all workspaces](/terraform/cloud-docs/users-teams-organizations/permissions#manage-all-workspaces) and [manage all projects](/terraform/cloud-docs/users-teams-organizations/permissions#manage-all-projects) permissions grant the workspace-level admin permission to every workspace in the organization. Organization-level permissions can only be managed by organization owners.

## Managing Workspace Access Permissions

> When a user creates a workspace, only [the owners team](/terraform/cloud-docs/users-teams-organizations/teams#the-owners-team), teams with the "Manage all workspaces" and/or the “Manage all projects” [organization permissions](/terraform/cloud-docs/users-teams-organizations/permissions#project-permissions), and teams with the “Project Admin” project permission can access a workspace with full admin permissions. You cannot override these teams' permissions through the workspace's specific permissions.

Reviewer: How would feel about restructuring this as a list? Right now it's starting to feel pretty dense, and I think a list would helps folks parse this a bit easier:
```suggestion
When a user creates a workspace, the following teams can access that workspace with full admin permissions:
- [the owners team](/terraform/cloud-docs/users-teams-organizations/teams#the-owners-team)
- teams with "Manage all workspaces" and/or “Manage all projects” [organization permissions](/terraform/cloud-docs/users-teams-organizations/permissions#project-permissions)
- teams with “Project Admin” project permissions
You cannot override these teams' permissions through the workspace's specific permissions.
```

@ritsok for thoughts on style!
